<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
   <class>Form</class>
   <widget class="QWidget" name="Form">
      <property name="geometry">
         <rect>
            <x>0</x>
            <y>0</y>
            <width>420</width>
            <height>350</height>
         </rect>
      </property>
      <property name="minimumSize">
         <size>
            <width>420</width>
            <height>350</height>
         </size>
      </property>
      <property name="windowTitle">
         <string>SmartDesk - Desktops verwalten</string>
      </property>
      <property name="window_type" stdset="0">
         <string>manage</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
            <number>10</number>
         </property>
         <property name="leftMargin">
            <number>20</number>
         </property>
         <property name="topMargin">
            <number>20</number>
         </property>
         <property name="rightMargin">
            <number>20</number>
         </property>
         <property name="bottomMargin">
            <number>20</number>
         </property>
         <item>
            <widget class="QLabel" name="label_title">
               <property name="font">
                  <font>
                     <family>Segoe UI</family>
                     <pointsize>14</pointsize>
                     <bold>true</bold>
                  </font>
               </property>
               <property name="text">
                  <string>Desktops verwalten</string>
               </property>
            </widget>
         </item>
         <item>
            <spacer name="verticalSpacer_top">
               <property name="orientation">
                  <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                  <size>
                     <width>20</width>
                     <height>10</height>
                  </size>
               </property>
            </spacer>
         </item>
         <item>
            <layout class="QHBoxLayout" name="contentLayout">
               <property name="spacing">
                  <number>15</number>
               </property>
               <item>
                  <!-- Linke Seite: Liste -->
                  <widget class="QScrollArea" name="scrollArea_desktops">
                     <property name="minimumSize">
                        <size>
                           <width>180</width>
                           <height>0</height>
                        </size>
                     </property>
                     <property name="widgetResizable">
                        <bool>true</bool>
                     </property>
                     <property name="frameShape">
                        <enum>QFrame::NoFrame</enum>
                     </property>
                     <widget class="QWidget" name="scrollAreaWidgetContents">
                        <property name="geometry">
                           <rect>
                              <x>0</x>
                              <y>0</y>
                              <width>180</width>
                              <height>200</height>
                           </rect>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_desktops">
                           <property name="spacing">
                              <number>5</number>
                           </property>
                           <property name="leftMargin">
                              <number>5</number>
                           </property>
                           <property name="topMargin">
                              <number>5</number>
                           </property>
                           <property name="rightMargin">
                              <number>5</number>
                           </property>
                           <property name="bottomMargin">
                              <number>5</number>
                           </property>
                           <!-- Hier werden die Buttons dynamisch per Python eingefügt -->
                        </layout>
                     </widget>
                  </widget>
               </item>
               <item>
                  <!-- Rechte Seite: Aktionen -->
                  <layout class="QVBoxLayout" name="actionsLayout">
                     <property name="spacing">
                        <number>8</number>
                     </property>
                     <item>
                        <widget class="QLabel" name="label_actions">
                           <property name="text">
                              <string>Aktionen:</string>
                           </property>
                           <property name="font">
                              <font>
                                 <family>Segoe UI</family>
                                 <pointsize>10</pointsize>
                                 <bold>true</bold>
                              </font>
                           </property>
                        </widget>
                     </item>
                     <item>
                        <widget class="QPushButton" name="btn_rename">
                           <property name="text">
                              <string>Umbenennen</string>
                           </property>
                           <property name="minimumSize">
                              <size>
                                 <width>0</width>
                                 <height>40</height>
                              </size>
                           </property>
                           <property name="enabled">
                              <bool>false</bool>
                           </property>
                           <property name="cursor">
                              <cursorShape>PointingHandCursor</cursorShape>
                           </property>
                        </widget>
                     </item>
                     <item>
                        <widget class="QPushButton" name="btn_wallpaper">
                           <property name="text">
                              <string>Hintergrund</string>
                           </property>
                           <property name="minimumSize">
                              <size>
                                 <width>0</width>
                                 <height>40</height>
                              </size>
                           </property>
                           <property name="enabled">
                              <bool>false</bool>
                           </property>
                           <property name="cursor">
                              <cursorShape>PointingHandCursor</cursorShape>
                           </property>
                        </widget>
                     </item>
                     <item>
                        <widget class="QPushButton" name="btn_more_options">
                           <property name="text">
                              <string>Mehr ...</string>
                           </property>
                           <property name="minimumSize">
                              <size>
                                 <width>0</width>
                                 <height>40</height>
                              </size>
                           </property>
                           <property name="enabled">
                              <bool>false</bool>
                           </property>
                           <property name="cursor">
                              <cursorShape>PointingHandCursor</cursorShape>
                           </property>
                        </widget>
                     </item>
                     <item>
                        <spacer name="verticalSpacer_actions">
                           <property name="orientation">
                              <enum>Qt::Vertical</enum>
                           </property>
                           <property name="sizeHint" stdset="0">
                              <size>
                                 <width>20</width>
                                 <height>40</height>
                              </size>
                           </property>
                        </spacer>
                     </item>
                  </layout>
               </item>
            </layout>
         </item>
         <item>
            <layout class="QHBoxLayout" name="bottomLayout">
               <item>
                  <spacer name="horizontalSpacer">
                     <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                        <size>
                           <width>40</width>
                           <height>20</height>
                        </size>
                     </property>
                  </spacer>
               </item>
               <item>
                  <widget class="QPushButton" name="btn_close">
                     <property name="text">
                        <string>Schließen</string>
                     </property>
                     <property name="minimumSize">
                        <size>
                           <width>100</width>
                           <height>35</height>
                        </size>
                     </property>
                     <property name="cursor">
                        <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                  </widget>
               </item>
            </layout>
         </item>
      </layout>
   </widget>
   <resources />
   <connections />
</ui>
